requirements.txt

```
@@ -1,6 +1,6 @@
 future
 gmsh
-matplotlib <= 3.7.0
+matplotlib <= 3.8.0
```

```suggestion
matplotlib
```
It is better not to fix version numbers unless we have to.